src/components/WwwFrame/Header/KvAtbModalContainer.vue

```diff
@@ -99,6 +125,25 @@ const fetchUserData = async () => {
 	});
 };
 
+const fetchPostCheckoutAchievements = async loanIds => {
+	await apollo.query({
+		query: postCheckoutAchievementsQuery,
+		variables: { loanIds }
+	}).then(({ data }) => {
+		const loanAchievements = data.postCheckoutAchievements?.overallProgress ?? [];
+		contributingAchievements.value = loanAchievements.filter(achievement => achievement.postCheckoutTier !== achievement.preCheckoutTier); // eslint-disable-line max-len
+		showModalContent.value = myKivaExperimentEnabled.value && contributingAchievements.value.length;
```

Comment:
You don't need to check `myKivaExperimentEnabled.value` here since it's checked where the method is called.